File: leetcode/Valid Anagram.cpp
class Solution {
public:
    bool isAnagram(string s, string t) {
            unordered_map<char,int>ans;
            for(int i=0;i<s.length();i++){
                ans[s[i]]++;
            }
            for(int i=0;i<t.length();i++){
                ans[t[i]]--;
            }
            for(auto &x:ans){
                if(x.second!=0){
                    return false;
                }
            }
            return true;
    }
};

File: leetcode/Valid anagram.py
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        mymap={}
        for char in s:
            mymap[char] = mymap.get(char,0)+1
        
        for i in t:
            mymap[i] = mymap.get(i,0)-1
        
        for i in mymap.values():
            if i!=0:
                return False
        
        return True
        
